README.md

```diff
@@ -1 +1,96 @@
 # xtractmime
+
+The `xtractmime` is a python module based on the complete implementation of 
+[MIME Sniffing Standards](https://mimesniff.spec.whatwg.org/). The module 
+provides simple function to determine the mime type or the content's format of the
+body in a web request or a web response by analyzing the byte sequences in it.
+
+Additionally, `xtractmime` provides various methods to determine if any mime type
+belongs to a specific mime group or not. For instance, HTML Mime types, XML Mime types
+and many more...
+
+---
+
+## Requirements
+
+* Python 3.6+
+
+---
+
+## Installation
+
+---
+
+## License
+
+`xtractmime` is distributed under a [BSD-3](https://opensource.org/licenses/BSD-3-Clause) license.
+
+---
```

Comment:
````suggestion
`xtractmime` is a [BSD-licensed](https://opensource.org/licenses/BSD-3-Clause)
Python 3.6+ implementation of the [MIME Sniffing
Standard](https://mimesniff.spec.whatwg.org/).

Install from [`PyPI`](https://pypi.python.org/pypi/xtractmime):

```
pip install xtractmime
```
````

- Keep the first paragraph short and to the point.
- Minimum Python version and license can be mentioned there, making later
  sections unnecessary while not making the paragraph much longer.
- API usage (i.e. mentioning exposed functions) is best covered on the “Basic 
  usage” section later.